xarray/tutorial.py

```diff
@@ -103,17 +92,75 @@ def open_dataset(
 
         url = f"{base_url}/raw/{version}/{path.name}"
 
-    _open = overrides.get(engine, _open_dataset)
     # retrieve the file
     filepath = pooch.retrieve(url=url, known_hash=None, path=cache_dir)
```

Comment:
with `known_hash=None` `pooch` will warn about the missing hash. Should I try to silence that?